domain_test.go

```diff
@@ -529,3 +543,28 @@ func TestDomainGetVcpusFlags(t *testing.T) {
 		t.Fatal("should have 1 cpu", num)
 	}
 }
+
+func TestQemuMonitorCommand(t *testing.T) {
+	dom, conn := buildTestQEMUDomain()
+	defer func() {
+        dom.Destroy()
+		dom.Undefine()
+		dom.Free()
+		conn.CloseConnection()
+	}()
+
+    if err := dom.Create(); err != nil {
+        t.Error(err)
+        return
+    }
+
+	if _, err := dom.QemuMonitorCommand(VIR_DOMAIN_QEMU_MONITOR_COMMAND_DEFAULT, "{\"execute\" : \"query-cpus\"}"); err != nil {
```

Comment:
@rgbkrk It's a nice idea, but we do want to avoid doing that. QMP's json doesn't have a formal "definition", except in source code and can change at will from one version to the next. It really should be application logic that's parsing it and maintaining it.